packages/ui/package.json

```diff
@@ -36,12 +36,12 @@
     "build:book": "vue-cli-service build --mode book",
     "lint": "vue-cli-service lint",
     "lint:style": "stylelint '**/*.{vue,html,css,scss}'",
-    "test:unit": "vue-cli-service test:unit -w 1",
-    "test:coverage": "vue-cli-service test:unit --coverage",
     "test:e2e": "vue-cli-service test:e2e",
+    "test:unit": "vitest",
+    "test:unit:coverage": "vitest --coverage --run",
```

Comment:
This makes a lot of junk and lie. 

Like, it says that template, css and comments are covered. Do we need it?
![image](https://user-images.githubusercontent.com/23530004/180570054-dcb7bc1a-8087-4417-a85d-d856d5b66106.png)

Reply:
Didn't really plan to do coverage so far. I'll remove it for now so that it doesn't create a visibility of working coverage solution.